src/interpreted/nodes.py

```diff
@@ -172,6 +172,23 @@ class Return(Statement):
     value: Expression
 
 
+@dataclass
+class alias(Statement):
```

Comment:
this doesn't need to inherit from `Statement`.

Since `alias` can't be used where a Statement can be used.